packages/gatsby-cli/src/recipes.ts

```diff
@@ -1,59 +1,17 @@
 import { trackCli } from "gatsby-telemetry"
-import execa from "execa"
-import * as path from "path"
-import * as fs from "fs"
-import detectPort from "detect-port"
+import runRecipe, { startGraphQLServer } from "gatsby-recipes"
```

Comment:
not really related to this PR but is there a way with the lock stuff to run a service in the foreground? Sometimes that's convenient to directly see the logs or to run the service with the node debugger.

Reply:
Yeah totally, _how_ you run your service is completely up to you 👍